bindings/dotnetcore/dotnet-core-binding/Microsoft.Azure.Devices.Gateway/Gateway.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.Devices.Gateway
+{
+    /// <summary>
+    ///  Wrapper for Azure IoT Gateway SDK. 
+    /// </summary>
+    public class Gateway
+    {
+        [DllImport(
+            "gateway",
+            CharSet = CharSet.Ansi,
+            EntryPoint = "Gateway_CreateFromJson",
+            CallingConvention = CallingConvention.Cdecl
+        )]
+        private static extern IntPtr CreateFromJsonInternal(string file_path);
+
+
+        /// <summary>
```

Comment:
Yeah, what I mean is, let's not duplicate the doxygen comments here. We already have the same doxygen comments (the sample JSON config) in gateway.h.

If someone goes to the doxygen pages for the C# gateway functions, they should see very minimal documentation that just refers to the C functions.